index.js

```diff
@@ -1,12 +1,9 @@
 // SPDX-FileCopyrightText: 2024 SmartRent
 //
 // SPDX-License-Identifier: Apache-2.0
-function handler(event) {
-    var request = event.request;
-    var headers = request.headers;
-    var querystring = request.querystring;
-    var now = (new Date()).toISOString();
-    var nonce = "";
+function handler({ request: { headers, querystring }) {
```

Comment:
```suggestion
function handler(event) {
    const headers = event.request.headers;
    const querystring = event.request.querystring;
```

It didn't seem happy with destructuring. Is this ok?